### PR TITLE
Contributing guide line

### PR DESCRIPTION
**Pull request by mahboob-awan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,33 +13,24 @@ between both the server and the client) for SEO reasons.
 [![Dependency Status](https://david-dm.org/quran/quran.com-frontend.svg)](https://david-dm.org/quran/quran.com-frontend) [![devDependency Status](https://david-dm.org/quran/quran.com-frontend/dev-status.svg)](https://david-dm.org/quran/quran.com-frontend#info=devDependencies)
 [![Code Climate](https://codeclimate.com/github/quran/quran.com-frontend.png)](https://codeclimate.com/github/quran/quran.com-frontend)
 
-## Getting started
-Simply clone this repo, then run `npm install` to install all the required node_modules.
-From there, you are ready to go!
+## How to contribute
+We trust that
+you will not steal this, this is at the end of the day for the sake of Allah and we
```
Comment (mmahalwy) [inline README.md]: thanks for putting this in, perhaps renaming 'stealing' to 'copying the code'. Stealing might be strong (especially that Islamically it is a huge sin and we don't want to blame anyone or accuse anyone of such a sin).

**Comment by mmahalwy** [comment]
thank you thank you thank you! This is awesome mashallah :)

File: README.md (hunk 1 above)
Comment (mahboob-awan) [inline README.md]: I think we should not worry based on few assumptions:
1 - A common person can't understand all this technology mess, so his idea of theft will not be successful 
2 - Only a professional can understand this project and professionals prefer to craft their own Art.

May ALLAH protect this project from all thieves and hackers. Ameen

**Comment by mahboob-awan** [comment]
i fix this soon, thanks for review 😊

File: README.md (hunk 1 above)
Comment (dashohoxha) [inline README.md]: I would suggest adding a license to the project, and I would suggest the GPL license. I wouldn't worry about people copying parts of the project, or even forking it and making it better, as long as their intentions are good, and God knows the intentions of every body.

**Comment by mahboob-awan** [comment]
@mmahalwy : i rebased the project, so U will see much difference in files.

**Comment by mmahalwy** [comment]
@mahboob-awan hmmm... seems like something happened there and shouldnt commit so many files. Can you either try rebasing again (`git pull --rebase origin develop`) or create another branch off of develop with just the CONTRIBUTING.md file?

**Comment by mahboob-awan** [comment]
@mmahalwy : sure, i create a new branch.

**Pull request closed without merging**